CHANGELOG.md

```diff
@@ -3,7 +3,8 @@
 See also https://github.com/neo4j/neo4j-python-driver/wiki for more details.
 
 ## NEXT RELEASE
-- ...
+- Renamed experimental `neo4j.RoutingControl.READERS` to `READ` and `WRITERS` to `WRITE`.
+- Renamed experimental `driver.query_bookmark_manager` to `execute_query_bookmark_manager`.
 
```

Comment:
Should you also add here statements about the apis which are not exp anymore?

Reply:
This changelog only contains breaking and major changes (see line 1).